src/ontology/mondo-edit.obo

```diff
@@ -435463,6 +435463,93 @@ synonym: "attenuated hypophosphatasia" EXACT [https://www.clinicalgenome.org/aff
 is_a: MONDO:0018570 {source="PMID:32973344", source="https://www.clinicalgenome.org/affiliation/40065/"} ! hypophosphatasia
 property_value: created_by http://orcid.org/0000-0002-5460-8025
 
+[Term]
+id: MONDO:0600014
+name: alveolar capillary dysplasia without misalignment
+def: "Alveolar capillary dysplasia (ACD) without misalignement of the pulmonary veins is a rare form of interstitial and vascular lung disease that presents as severe pulmonary hypertension and refractory hypoxemia early in life." [https://orcid.org/0000-0003-0113-912X, https://www.sciencedirect.com/topics/medicine-and-dentistry/alveolar-capillary-dysplasia]
+synonym: "ACD without misalignment" EXACT ABBREVIATION []
+is_a: MONDO:0020295 {source="https://orcid.org/0000-0003-0113-912X"} ! congenital pulmonary veins anomaly
+property_value: created_by http://orcid.org/0000-0002-5460-8025
+
+[Term]
+id: MONDO:0600015
+name: alveolar capillary dysplasia with misalignment
+def: "Alveolar capillary dysplasia (ACD) with misalignment of the pulmonary veins is a rare form of interstitial and vascular lung disease that presents as severe pulmonary hypertension and refractory hypoxemia early in life." [https://orcid.org/0000-0003-0113-912X, https://www.sciencedirect.com/topics/medicine-and-dentistry/alveolar-capillary-dysplasia]
+synonym: "ACD with misalignment" EXACT ABBREVIATION []
+is_a: MONDO:0020295 {source="https://orcid.org/0000-0003-0113-912X"} ! congenital pulmonary veins anomaly
+property_value: created_by http://orcid.org/0000-0002-5460-8025
+
+[Term]
+id: MONDO:0600016
+name: acinar dysplasia
+def: "Congenital acinar dysplasia (AcDys) is a lethal, developmental lung malformation resulting in neonatal respiratory insufficiency. This entity is characterized by pulmonary hypoplasia and arrest in the pseudoglandular stage of development, resulting in the absence of functional gas exchange." [https://orcid.org/0000-0003-0113-912X, PMID:31641660]
```

Comment:
Avoid repeating the term name in the definition 

and if the term name is repeated, make sure it it is consistent with the actual term name!

e..g is this congenital AcDys? is there a non-congenital AcDys?

https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/